streams/src/test/java/org/apache/kafka/streams/state/internals/AbstractKeyValueStoreTest.java

```diff
@@ -225,6 +277,7 @@ public void testSize() {
         store.put(2, "two");
         store.put(4, "four");
         store.put(5, "five");
+        store.flush();
```

Comment:
Some stores are caches. Their `approximateNumEntries`currently returns the items in the underlying store. Without flushing the underlying store would have no items.